app/views/errorAlert.scala.html

```diff
@@ -8,8 +8,8 @@ <h3 id="alert-title">Error</h3>
         <div class="alert-text standalone-text">
             It looks like you're using NetLogo Web in standalone mode.
             <br />
-            If the above error is being caused by a missing primitive, we recommend a quick visit to
-                <a href="http://netlogoweb.org" target="_blank">NetLogoWeb.org</a>.
+            If the above error is being caused by an unknown primitive, we recommend a quick visit to
```

Comment:
I feel like "unknown" isn't the correct word for it.  Seems to me that it **is** known, or else they wouldn't be getting this error.  Perhaps it would be better to use the term "unimplemented" or "not yet implemented"?